app/src/main/res/layout/activity_info.xml

```diff
@@ -20,72 +20,72 @@
                 android:id="@+id/ll_info"
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
-                android:layout_margin="10dp"
+                android:layout_margin="@dimen/margin1"
```

Comment:
do not use it as `margin1`. it has to be generalized
Not only this but all the other also